packages/example-react/package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "example-react",
   "private": true,
-  "version": "1.0.0-beta.5",
+  "version": "1.0.0-icons-remake.1",
```

Comment:
I'm not sure how to properly release this change as it seems to introduce breaking changes.
We should either release it as alpha, consider new package, or deliver the change in steps, allowing both deprecated form and the new one.